### PR TITLE
add new OMIM terms (OMIM migration)

### PR DESCRIPTION
**Pull request by nicolevasilevsky**
this adds new OMIM terms

note, I did not add terms to the exclusion list yet or deal with the new OMIMPS
[spreadsheet here](https://docs.google.com/spreadsheets/d/1RPZY5fEmO5zssrq0JjonB8HrGXdl3E2lUH5I5xPYFew/edit#gid=1773061707)

### CODE REVIEW

**Comment by nicolevasilevsky** [comment]
FYI- 
I created the new term

[Term]
id: MONDO:0100526
name: breast-ovarian cancer, familial, susceptibility to
xref: OMIMPS:604370 {source="MONDO:equivalentTo"}
is_a: MONDO:0020573 {source="OMIMPS:604370"} ! inherited disease susceptibility
relationship: has_characteristic MONDO:0021152 {source="OMIMPS:604370"} ! inherited
property_value: http://purl.org/dc/terms/creator https://orcid.org/0000-0001-5208-3432

This was previously a Mondo term (MONDO:0000163) and was obsoleted and merged with ‘hereditary breast ovarian cancer syndrome’. I reinstated it as a child of ‘inherited disease susceptibility’.

I do not think OMIMPS https://omim.org/phenotypicSeries/PS604370 is equiv to ‘hereditary breast ovarian cancer syndrome’ 

The OMIMPS does not match the def: An autosomal dominant inherited syndrome caused by mutations in the BRCA1 or BRCA2 genes. Patients are at high risk of developing breast cancer, particularly before the age of fifty, high risk of developing a second primary breast cancer, and high risk of developing both breast and ovarian cancer.

the OMIMPS includes classes that are not just BRCA.

File: src/ontology/mondo-edit.obo [hunk 1]
```diff
@@ -464825,6 +464835,487 @@ name: neurological muscular channelopathy due to a genetic ryanodine receptor de
...
+name: autoinflammatory disease, systemic, with vasculitis
+xref: OMIM:620376 {source="MONDO:equivalentTo"}
+is_a: MONDO:0003847 {source="OMIM:620376", source="https://orcid.org/0000-0001-5208-3432"} ! hereditary disease
+is_a: MONDO:0019751 {source="https://orcid.org/0000-0001-5208-3432"} ! autoinflammatory syndrome
+
+[Term]
+id: MONDO:0957273
+name: Charcot-Marie-Tooth disease, dominant intermediate A
+xref: OMIM:620378 {source="MONDO:equivalentTo"}
+is_a: MONDO:0003847 {source="OMIM:620378"} ! hereditary disease
```
Comment (sabrinatoro) [inline src/ontology/mondo-edit.obo:465074]: Could the parent be 
MONDO:0019548
'autosomal dominant intermediate Charcot-Marie-Tooth disease'

File: src/ontology/mondo-edit.obo [hunk 2]
```diff
@@ -464825,6 +464835,487 @@ name: neurological muscular channelopathy due to a genetic ryanodine receptor de
...
+id: MONDO:0957278
+name: oocyte/zygote/embryo maturation arrest 20
+xref: OMIM:620383 {source="MONDO:equivalentTo"}
+is_a: MONDO:0014769 {source="OMIM:620383"} ! inherited oocyte maturation defect
+
+[Term]
+id: MONDO:0957279
+name: auditory neuropathy, autosomal dominant 2
+xref: OMIM:620384 {source="MONDO:equivalentTo"}
+is_a: MONDO:0003847 {source="OMIM:620384"} ! hereditary disease
```
Comment (sabrinatoro) [inline src/ontology/mondo-edit.obo:465092]: could parent be: 
MONDO:0021944
'auditory neuropathy'

File: src/ontology/mondo-edit.obo [hunk 3]
```diff
@@ -464825,6 +464835,487 @@ name: neurological muscular channelopathy due to a genetic ryanodine receptor de
...
+id: MONDO:0957301
+name: spermatogenic failure 84
+xref: OMIM:620409 {source="MONDO:equivalentTo"}
+is_a: MONDO:0004983 {source="OMIM:620409"} ! spermatogenic failure
+
+[Term]
+id: MONDO:0957302
+name: low density lipoprotein cholesterol level quantitative trait locus 3
+xref: OMIM:620410 {source="MONDO:equivalentTo"}
+is_a: MONDO:0003847 {source="OMIM:620410"} ! hereditary disease
```
Comment (sabrinatoro) [inline src/ontology/mondo-edit.obo:465128]: it looks like this term should be excluded as it is not a disease.

**Review by sabrinatoro — CHANGES_REQUESTED** [review]
@nicolevasilevsky I have a few suggestions for more specific parents and one term that I think should be excluded. 
Thank you for taking care of this.

File: src/ontology/mondo-edit.obo (hunk 1 above)
Comment (nicolevasilevsky) [reply src/ontology/mondo-edit.obo:465074]: done

File: src/ontology/mondo-edit.obo (hunk 3 above)
Comment (nicolevasilevsky) [reply src/ontology/mondo-edit.obo:465128]: good catch, thanks

File: src/ontology/mondo-edit.obo (hunk 2 above)
Comment (nicolevasilevsky) [reply src/ontology/mondo-edit.obo:465092]: done

**Pull request merged**